backends/aerospike_test.go

```diff
@@ -253,7 +253,7 @@ func TestClientPut(t *testing.T) {
 			desc: "AerospikeBackend.Put() does not throw error",
 			inAerospikeClient: &goodAerospikeClient{
 				records: map[string]*as.Record{
-					"defaultKey": &as.Record{
+					"defaultKey": {
```

Comment:
Makes perfect sense to remove `&as.Record`. Good catch

Reply:
That came from adding the simplification flag to gofmt, as we do for PBS. Does make sense though, I agree.